src/workloads/transactions/templates/LLTMixedLowLtc.tmpl

```diff
@@ -5,8 +5,8 @@ Description: |
 
 GlobalDefaults:
   # Start: The following block gets updated for the other configurations.
-  # Not In-memory: Database size works out about 30GB.
-  InitialDocumentCount: &InitialNumDocs 49000000
+  # In-memory: Database size works out about 12GB.
```

Comment:
This is odd but I think it is just git is just making assumptions about what files were moved (the values are actually correct).

Also we could delete these files, but I wanted to keep them around in case we ever want to deploy other workloads.